Mongo db credentials update

gandreini opened the pull request:
I added some updates on how to configure Mongo DB Atlas.
Currently added to a section: **Connecting to Mongo DB Atlas**
I don't know Mongo DB very well and I'm not sure if these guidelines apply also to other versions of Mongo DB.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/n8n-io/n8n-docs?pullRequest=1490) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/n8n-io/n8n-docs?pullRequest=1490) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/n8n-io/n8n-docs?pullRequest=1490) it.</sub>

gandreini commented:
I replaced "using OAuth" with "Using Connection String or Values" and added some generic information about how to connect to any type of MongoDB. Added link to connection string details.

StarfallProjects reviewed (changes requested):
@gandreini couple of things you could quickly fix:

- UI text should be bold but not italix
- Some of the screenshots are unclear/details too small to read.
- All info must be in text as well as screenshot (accessibility)
